docs/deploy/security_groups.md

```diff
@@ -1,32 +1,58 @@
 
-The controller flag `--enable-backend-security-group` (default `true`) is used to enable/disable the shared backend security group. The flag `--backend-security-group` (default empty) is used to pass in the security group to use as a shared backend security group. If it is empty, the LBC will auto-generate a security group with the following name and tags -
+- When configuring security group rules at the ENI/Instance level, use the Security Group ID of the backend security group.
 
-```
-name: k8s-traffic-<cluster_name>-<hash_of_cluster_name>
-tags: 
-    elbv2.k8s.aws/cluster: <cluster_name>
-    elbv2.k8s.aws/resource: backend-sg
-```
+### Configuration
 
-You can turn off the shared backend security group feature by setting `--enable-backend-security-group` to `false`. However, if you have a high number of Ingress resources with frontend security groups auto-generated by the controller, you might run into security group rule limits on the instance/ENI security groups.
+Control and manage the backend security groups with the controller flags:
+
+- **Enable or Disable:** Use `--enable-backend-security-group` (default `true`) to toggle the shared backend security group.
+
+- **Specification:** If `--backend-security-group` (default empty) is left empty, an auto-generated security group with the following attributes will be created:
+
+  ```yaml
+  name: k8s-traffic-<cluster_name>-<hash_of_cluster_name>
+  tags: 
+      elbv2.k8s.aws/cluster: <cluster_name>
+      elbv2.k8s.aws/resource: backend-sg
+  ```
+
+- **Considerations:**
+  - Be cautious when having numerous Ingress resources with auto-generated frontend security groups as it may lead to security group rule limits on the instance/ENI security groups.
+  - Ensure `--enable-backend-security-group` is set to true when specifying `alb.ingress.kubernetes.io/manage-backend-security-group-rules` or `service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules`, to avoid errors.
 
 ### Management of Backend Security Group Rules
 
-When the LBC auto-creates the frontend security group for a load balancer, it automatically adds the security group rules to allow traffic from the load balancer to the backend instances/ENIs.
+Understand how to configure and manage security group rules effectively:
+
+- **Auto-Creation:** When LBC auto-creates the frontend security group, it automatically adds rules allowing traffic from the load balancer to the backend instances/ENIs.
+- **Specification:** Use specific annotations on Ingress or Service resources for specifying frontend security groups, and manage the rules of instance/ENI security groups via additional annotations.
```

Comment:
This mangles the section to remove its entire purpose. The point is that specifying the frontend groups will disable LBC's behavior of adding security group rules to the security groups of targets. Furthermore, this behavior can be re-enabled with an annotation, which one depending on whether the resource is Ingress or Service.